hbase-client/src/main/java/org/apache/hadoop/hbase/quotas/QuotaRetriever.java

```diff
@@ -52,21 +50,9 @@ public class QuotaRetriever implements Closeable, Iterable<QuotaSettings> {
   private Connection connection;
   private Table table;
 
-  /**
-   * Should QutoaRetriever manage the state of the connection, or leave it be.
-   */
-  private boolean isManagedConnection = false;
-
   QuotaRetriever() {
```

Comment:
What a weird API. This class is decorated as `IA.Public` but it's only created via these static factory method? Any what's with using the default constructor + an `init` method -- what happened to RAII ?’

Reply:
Only public methods.

Reply:
Okay, then this PR is complying with the deprecation policy now